src/main/scala/mesosphere/marathon/api/akkahttp/Directives.scala

```diff
@@ -76,4 +76,30 @@ object Directives extends AuthDirectives with LeaderDirectives with AkkaDirectiv
       case Success => f(Unit)
     }
   }
+
+  def extractTaskKillingMode: Directive1[TaskKillingMode] =
+    (parameter("scale".as[Boolean].?(false)) & parameter("wipe".as[Boolean].?(false))).tflatMap {
```

Comment:
What is `tflatMap`?

Reply:
A something like `flatMap` but for directives :)

Reply:
I did not know that one either. Thanks for asking and answering :)

Reply:
I actually stumbled upon this as well.